neo4j/pom.xml

```diff
@@ -130,4 +156,25 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
+
+    <profiles>
```

Comment:
Can you check what @mp911de does in the pending Redis PR? There two Surefire executions are configured. Would be nice to do the same here.